demisto_sdk/commands/validate/validators/MR_validators/MR101_validate_empty_keys.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = ModelingRule
+
+
+class ValidateEmptyKeysValidator(BaseValidator[ContentTypes]):
+    error_code = "MR101"
+    description = (
+        "Validate that the modeling rules keys - rules and schema exist and are empty"
+    )
+    rationale = "This standardization is to handle modeling rules correctly."
```

Comment:
Rationale should explain why this validation is needed, i.e. why we want to keep the schema and modeling rules key empty. You can sync with the siem team or check the original ticket requesting the v validation to better understand this.